### PR TITLE
Group 3

### PR DESCRIPTION
**Pull request by AlenaSutiagina**

### CODE REVIEW

File: src/main/java/de/cas/mse/exercise/SumDigPower.java [hunk 1]
```diff
@@ -5,20 +5,16 @@
...
-		List<Long> result = new ArrayList<Long>();
+		List<Long> numbersThatMatch = new ArrayList<Long>();
 
-		for (long i = a; i < b; i++) {
-
-			List<Long> longs = new ArrayList<Long>();
+		for (long i = rangeStart; i < rangeEnd; i++) {
 
+			
 			String temp = Long.toString(i);
```
Comment (Harzenetter) [inline src/main/java/de/cas/mse/exercise/SumDigPower.java:15]: what is temp? temprature perhaps ;-)?

File: src/main/java/de/cas/mse/exercise/SumDigPower.java [hunk 2]
```diff
@@ -5,20 +5,16 @@
...
 
-			// Split values
-			for (int j = 0; j < temp.length(); j++) {
-				longs.add(Long.valueOf(temp.substring(j, j + 1)));
-			}
+			List<Long> longs = extractDigits(temp);
 
 			// Create sum
 			long sum = 0;
```
Comment (The-Drummer6) [inline src/main/java/de/cas/mse/exercise/SumDigPower.java:20]: extracting a method could be helping to make this part more readable

File: src/main/java/de/cas/mse/exercise/SumDigPower.java [hunk 3]
```diff
@@ -5,20 +5,16 @@
 
 public class SumDigPower {
 
-	public List<Long> sumDigPow(long a, long b) {
+	public List<Long> sumDigPower(long rangeStart, long rangeEnd) {
```
Comment (Harzenetter) [inline src/main/java/de/cas/mse/exercise/SumDigPower.java:8]: Won't run without changing tests

File: src/main/java/de/cas/mse/exercise/SumDigPower.java [hunk 4]
```diff
@@ -28,14 +24,21 @@
 
 			// Test if sum is equal
```
Comment (The-Drummer6) [inline src/main/java/de/cas/mse/exercise/SumDigPower.java:25]: think about extracting methods for all inline comments -> then the code is more self explaining

File: src/main/java/de/cas/mse/exercise/SumDigPower.java [hunk 5]
```diff
@@ -28,14 +24,21 @@
...
 
 		}
 
-        System.out.println(result);
-		return result;
+        System.out.println(numbersThatMatch);
+		return numbersThatMatch;
 
 	}
+        private List<Long> extractDigits(String temp) {
```
Comment (Harzenetter) [inline src/main/java/de/cas/mse/exercise/SumDigPower.java:36]: temp again?

**Pull request merged**